visualizer/src/Controls/ImageControls/RawControls/ColorModeToggle.js

```diff
@@ -0,0 +1,63 @@
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Switch from '@material-ui/core/Switch';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
+import { useSelector } from '@xstate/react';
+import React, { useEffect, useRef } from 'react';
+import { useImage, useRaw } from '../../../ServiceContext';
+
+function ColorModeToggle() {
+  const image = useImage();
+  const grayscale = useSelector(image, state => state.context.grayscale);
+  const raw = useRaw();
+  const { send } = raw;
+
+  // Adds mousetrap class so hotkeys work after using switch
+  const inputRef = useRef();
+  useEffect(() => {
+    const input = inputRef.current;
+    input.className = input.className + ' mousetrap';
```

Comment:
Prefer the string formatting over concatenation:

```suggestion
    input.className = `${input.className}  mousetrap`;
```